__tests__/server/plugins/addFrameOptionsHeader.spec.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import addFrameOptionsHeader from '../../../src/server/plugins/addFrameOptionsHeader';
+import { updateCSP } from '../../../src/server/plugins/csp';
+
+beforeEach(() => {
+  updateCSP('');
+});
+
+describe('empty frame-ancestors', () => {
+  test('does not add frame options header', () => {
```

Comment:
in the `addCacheHeaders.spec.js` test file we use `it`, and here we are using `test`. We should be consistent